src/test/java/uk/gov/companieshouse/api/accounts/interceptor/TransactionInterceptorTest.java

```diff
@@ -31,6 +31,11 @@
 @TestInstance(Lifecycle.PER_CLASS)
 public class TransactionInterceptorTest {
 
+    private static final String ACCOUNTS_ID = "1234561";
+    private static final String ACCOUNTS_API_END_POINT = "http://public/transactions/%s/company-accounts/%s";
+    private static final String FILING_GENERATOR_END_POINT = "http://private/transactions/%s/company-accounts/%s/filings";
+    private static final String TRANSACTION_ID = "1234561-1234561-1234561";
```

Comment:
Could these be changed at all?

If you concatenated `ACCOUNTS_ID` and `TRANSACTION_ID` to `ACCOUNTS_API_END_POINT` when declaring the `String` (as opposed to concatenating  on each `get` call), the `FILING_GENERATOR_END_POINT` could become 

`private static final String `FILING_GENERATOR_END_POINT = ACCOUNTS_API_END_POINT + "/filings"'

By doing this, we could also remove the need for the getters.

Reply:
It is different approach but either way it works. I could change it but i dont see the benefit on it.